docs/getting-started.md

```diff
@@ -139,8 +156,28 @@ In the **Examples** section of our documentation, you can find links to several
 * [OpenUI5](https://github.com/SAP/luigi/tree/master/core/examples/luigi-example-openui5)
 * [Svelte](https://github.com/SAP/luigi/tree/master/core/examples/luigi-example-svelte)
 * [Plain JavaScript](https://github.com/SAP/luigi/tree/master/core/examples/luigi-example-js)
+* [NextJS](https://github.com/SAP/luigi/tree/master/core/examples/luigi-example-next)
 
 You can install them by following the instructions in the `README` file of each example.
 
-**Advanced users** can install our end-to-end test sample application which includes all of Luigi's features in one place:
-* [e2e example](https://github.com/SAP/luigi/tree/master/test/e2e-test-application)
+### Luigi shopping app (tutorial)
+
+Our [tutorial](https://developers.sap.com/group.luigi-app.html) on how to create a React and UI5 web-shopping application is intended for beginners, but it still delves deeper into Luigi's functions. It covers topics such as:
+- how to create a Luigi app from scratch
+- how to use the Luigi Core and Client APIs
+- how to create a micro frontend
+- how to add localization to your app and display it in multiple languages
+
+The whole tutorial should take about an hour to complete. The source code for the tutorial app can be found [here](https://github.com/SAP-samples/luigi-micro-frontend-application).
+
+### e2e example
+
+This example application was created for testing purposes and it includes all possible Luigi features in one place. This makes it complicated and unsuitable for beginners, but it can be useful if you want to explore our framework in more detail or [contribute](https://github.com/SAP/luigi/blob/master/CONTRIBUTING.md) to the Luigi project.
```

Comment:
```suggestion
This example application was created for testing purposes and it includes all possible Luigi features in one place. It might be complicated and unsuitable for beginners, but useful if you want to explore our framework in more detail or [contribute](https://github.com/SAP/luigi/blob/master/CONTRIBUTING.md) to the Luigi project.
```